### ISSUE TITLE
Classname in Snippets broken after update.

### ISSUE THREAD

**Issue by malarinv** [atom/language-python#249]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Snippet for class,property are generating invalid code.

### Steps to Reproduce

1. create a empty python file with atom
2. type `class[tab][tab]`

**Expected behavior:**
It should have generated the following
```
class ClassName|(object):
    """docstring for ClassName|."""
    def __init__(self, arg):
        super(ClassName|, self).__init__()
        self.arg = arg
```
**Actual behavior:**
generates the following
```
class ClassName|(object):
    """docstring for [object Object]."""
    def __init__(self, arg):
        super([object Object], self).__init__()
        self.arg = arg
```
**Reproduces how often:**
Always
### Versions
Atom    : 1.25.1
Electron: 1.7.11
Chrome  : 58.0.3029.110
Node    : 7.9.0

apm  1.19.0
npm  3.10.10
node 6.9.5 x64
atom 1.25.1
python 2.7.14
git 2.10.1

**Comment by Arcanemagus** [comment]
It looks like this regressed since Atom v1.24.1, as this is what you get in that version:
![image](https://user-images.githubusercontent.com/427137/38434211-66cbe092-3982-11e8-9a42-ce4b96c0bd35.png)

**Comment by lee-dohm** [comment]
May be related to https://github.com/atom/snippets/pull/267. @Arcanemagus can you double check this to see if it would be fixed by that PR?

**Comment by Arcanemagus** [comment]
I can confirm that a version of Atom that includes that PR works fine when inserting that snippet 🎉.

This has already been fixed in https://github.com/atom/snippets/pull/267. It was merged into Atom in https://github.com/atom/atom/commit/067add71f8147d3864e3885eadecf0f7052ca299 😉.

Thanks for reporting this!

**Comment by Arcanemagus** [comment]
This has been released in Atom v1.26.0-beta2.
